recipes/py-trees/meta.yaml

```
@@ -0,0 +1,76 @@
+{% set name = "py-trees" %}
+{% set version = "2.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/py_trees-{{ version }}.tar.gz
+  sha256: 8a2024ca6cb966c3c571d91e2d60c90cc562c4d237a0af8ab7a7c27c2cff0795
```

```suggestion
  sha256: cd8738a80e7422aec2b133270b6aa92e3e83926ded2ba9b3614028dba99cdb41
```
Update the SHA256 hash when bumping the version @naveedhd